code/modules/mob/living/carbon/human/species.dm

```diff
@@ -114,10 +114,10 @@
 
 
 //Please override this locally if you want to define when what species qualifies for what rank if human authority is enforced.
-/datum/species/proc/qualifies_for_rank(rank, list/features)
+/datum/species/proc/does_not_qualify_for_rank(rank, list/features)
 	if(rank in command_positions)
-		return 0
-	return 1
+		return "NON-HUMAN"
+	return null
```

Comment:
I think an empty string would return false.

I also think this proc should be renamed, the name indicates that it returns a true/false statement.